misc/scripts/install-local.sh

```diff
@@ -247,7 +248,7 @@ if [[ -z $PACSTALL_REMOVE ]] && [[ -z $PACSTALL_INSTALL ]]; then
 	rm -rf '"$name"' 2> /dev/null
 	hash -r
 	if declare -F removescript >/dev/null ; then
-		removescript
+		(set -euo pipefail; removescript)
```

Comment:
Change the syntax to the `bash -ce "removescript"`, and make sure to export the function beforehand.